### DISCUSSION TITLE
怎么去除中间的空白页面

### DISCUSSION THREAD

**Discussion by phoenixnir** [TheNetAdmin/zjuthesis#323] [General]
学院给出的模板之中，各个part之间没有空白页面，怎么在保持页码正确的情况下删去空白页面？

**Comment by TheNetAdmin** [comment]
https://github.com/TheNetAdmin/zjuthesis/blob/279744d56291273f93b1e9510d6c6c5f69421f1f/zjuthesis.tex#L20

改为false
